From: Michael Froman <mfroman@mozilla.com>
Date: Mon, 20 Jun 2022 22:34:00 +0000
Subject: Bug 1921706 - simplify removing libatomic on linux builds.
 r=ng,webrtc-reviewers DONTBUILD

Note: this produces no changes in the generated moz.build files.

o  changeset:   597701:a26d44969a83
~  user:        Michael Froman <mfroman@mozilla.com>
   date:        Mon Jun 14 17:39:29 2021 -0500
   files:       third_party/libwebrtc/build/config/linux/BUILD.gn
   description:
   Bug 1654112 - don't use libatomic on linux builds. r=ng

In moz-central a26d44969a83 I added an include of webrtc.gni for
the build_with_mozilla flag, but there is already a Mozilla
modifcation in this file that does not rely on that flag.  This
simplification will help with future google build directory move.

Differential Revision: https://phabricator.services.mozilla.com/D224069
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/d57bb9cfa9a7679fa5d3f47ae9343928700ec103
---
 config/linux/BUILD.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/linux/BUILD.gn b/config/linux/BUILD.gn
index cc4749ea0..b96e8cbe3 100644
--- a/config/linux/BUILD.gn
+++ b/config/linux/BUILD.gn
@@ -40,10 +40,13 @@ config("runtime_library") {
     defines = [ "OS_CHROMEOS" ]
   }
 
+  # Mozilla disable libatomic on linux builds
+  if (false) {
   if ((!is_chromeos || default_toolchain != "//build/toolchain/cros:target") &&
       (!use_custom_libcxx || current_cpu == "mipsel")) {
     libs = [ "atomic" ]
   }
+  }
 }
 
 config("libcap") {
@@ -54,6 +57,7 @@ config("libresolv") {
   libs = [ "resolv" ]
 }
 
+# Mozilla disable glib - Bug 1654112 (hg sha 127ace4d8887)
 if (false && use_glib) {
   pkg_config("glib") {
     packages = [
